packages/fxa-settings/src/components/BrandMessaging/index.tsx

```diff
@@ -68,37 +68,40 @@ const BrandMessaging = ({
   }
 
   return (
-    <div id="banner-brand-message" className="fixed w-full top-0 left-0">
+    <div
+      id="banner-brand-message"
+      className="w-full relative mobileLandscape:absolute mobileLandscape:top-0 mobileLandscape:left-0"
+    >
       <div className="flex relative justify-center p-2 brand-banner-bg">
         {mode === 'prelaunch' && (
           <div className="flex" data-testid="brand-prelaunch">
-            <div className="m-logo flex-none relative mt-1 mr-4 mb-2">
-              <div className="bg-black">
-                <Localized id="brand-m-logo" attrs={{ alt: true }}>
-                  <img
-                    src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABwAAAAQCAYAAAAFzx/vAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAEKSURBVHgB7ZThDYIwEIWLcQBHYAPdQDfQEdxAR9AJdAN0AnECYAOYoGwAG5yv4UiacrX9ofGPL3kEel/v2tI2IaKFUmqnpiqTJGk5voHXsHnv4SdipQ2DSznPkpkGzG2S1YAkaw9vYO2JX6wcBw+jeSBRBTMK68QDeydNwyoFC47q4AdceGKa3wvmtMAd7YILHuVRAB/26Mg/m5Uzgc6JZ7H/MhU4HUqGtqvD5GNspvyqzS4V2t22RurrfHcxBXsVp1guWPAr+hf8uObmgW17UsM9aSvldqMeO/bqybHl45ODqT3MinMNcQpLjz1JvnGM9hbjuyCy3ywpdA5w9lm7w5XA2MvZwjeBqV7FRzhNrCW4kwAAAABJRU5ErkJggg=="
-                    alt="Mozilla m logo"
-                  />
-                </Localized>
-              </div>
+            <div className="flex-none relative">
+              <Localized id="brand-m-logo" attrs={{ alt: true }}>
+                <img
+                  className="w-8 h-8 bg-black m-4 mt-1"
+                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzIiIGhlaWdodD0iMzIiIHZpZXdCb3g9IjAgMCAzMiAzMiIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHJlY3QgeD0iMC43NjkyMjYiIHk9IjAuNzY5MjI2IiB3aWR0aD0iMzAuNDYxNSIgaGVpZ2h0PSIzMC40NjE1IiBmaWxsPSJibGFjayIvPgo8cGF0aCBkPSJNMjcuNzMxNCAyMC41MDg0SDI5LjY0NDFWMjMuNjE1NEgyMy42MTIxVjE1LjI3NDVDMjMuNjEyMSAxMi42NTY3IDIyLjc3ODEgMTEuNjcyOCAyMS4xNTkyIDExLjY3MjhDMTkuMTk3MiAxMS42NzI4IDE4LjQxMjQgMTMuMTYwOSAxOC40MTI0IDE1LjIxMzlWMjAuNDQ5NkgyMC4zMjUxVjIzLjU2MDRIMTQuMzAwOFYxNS4yNjg5QzE0LjMwMDggMTIuNjUxIDEzLjQ2NjcgMTEuNjY3MSAxMS44NDc4IDExLjY2NzFDOS44ODU4MyAxMS42NjcxIDguNzg4MjUgMTMuMTU1MiA4Ljc4ODI1IDE1LjIwODJWMjAuNDQzOUgxMS44MzY0VjIzLjU2MDRIMy4wMTQxNFYyMC40NTM0SDQuODIyNThWMTIuMzU3MkgyLjkyNjk0VjguNjU0OThIOC4zNzY5VjEwLjk5NDJDOS41MDExOSA5LjMyNDQ0IDExLjM5OTMgOC4zNDMyMiAxMy40MTE3IDguMzkxNDlDMTUuNTIxNCA4LjI3NjQ3IDE3LjQzOTkgOS42MDg1NCAxOC4wNjkzIDExLjYyNTRDMTguNzQ2NyA5LjY0MzQ2IDIwLjYzNSA4LjMzMjg3IDIyLjcyODggOC4zOTE0OUMyNC4xMDE5IDguMzI4NzkgMjUuNDM1MSA4Ljg2MzI2IDI2LjM4NDYgOS44NTcxNkMyNy4zMzQyIDEwLjg1MTEgMjcuODA3MyAxMi4yMDcyIDI3LjY4MjEgMTMuNTc2MVYyMC41MDg0SDI3LjczMTRaIiBmaWxsPSJ3aGl0ZSIvPgo8L3N2Zz4K"
+                  alt="Mozilla m logo"
```

Comment:
Missing l10n here on this alt text too.

Reply:
What else do I have to do? 

I've wrapped it with Localized and alt:true? The brand-m-logo is in the ftl file.

Reply:
With a Localized/FtlMsg wrapper and alt attribute, the ftl string has to be like so to be picked up:

```
# Alt text for 'm' logo in banner header
brand-m-logo = 
  .alt = Mozilla m logo
```

Alternative is to use `ftlMsgResolver` to directly set the localized string as a variable in the alt attribute.

Reply:
Adjusted ftl file.